### PR TITLE
Propcheck human

### PR DESCRIPTION
**Pull request by mattsmee**
- randomized messages is not in this PR
- `human` doesn't take optional custom messages in this PR

### CODE REVIEW

**Comment by jonajonlee** [comment]
looks good to me!

:shipit:

**Comment by slambertftw** [comment]
woohoo! :shipit:

**Comment by jonajonlee** [comment]
:shipit: 

image

File: lib/human.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    formattedUserInput
+  , formattedIncorrectInput
+  , formattedExpectedInput
+  , formattedCorrection
+  ])
+));
+
+
+// human :: ( Spec, PropCheckResult ) -> String
+const human = (spec, result) => {
```
Comment (scull7) [inline lib/human.js:114]: why not... 

``` javascript
// human :: ( Spec, PropCheckResult ) -> Maybe String
const human = _.curryN(2, _.ifElse(nothingIsWrong, _.always(null), readableErrorMessage);
```

**Pull request merged**